cmd/opentelemetry/app/exporter/badgerexporter/factory.go

```diff
@@ -91,17 +91,17 @@ func (f Factory) CreateDefaultConfig() configmodels.Exporter {
 
 // CreateTraceExporter creates Jaeger Cassandra trace exporter.
 // This function implements OTEL component.ExporterFactory interface.
-func (f Factory) CreateTraceExporter(
+func (f Factory) CreateTracesExporter(
```

Comment:
hehe, what happens when so many people contributing are not native speakers. "trace exporter" was grammatically correct name.

Reply:
not a jab at you, @Vemmy124 , but on the upstream change

Reply:
I confess it did sound strange to my ears, but I'm not a native speaker myself, so, thought it was a problem with my ears :-)

Reply:
I believe that was bogdandrutu's idea https://github.com/open-telemetry/opentelemetry-collector/issues/1974

Reply:
And maybe that's really the problem of being not native speaker, but I agree with him, because the component consumes the structure named pdata.Traces

Reply:
It doesn't matter what exact data type is passed, a component that exports traces (plural) is "trace exporter" (noun used as adjective in singular form).